andela/activo-web
# #159348904 Create New Centers

**What does this PR do?**
It enables a user to create a new center

**Description of Task to be completed?**
- create a form to enable users to create new centers
- implement cloudinary image upload
- implement image preview

**How should this be manually tested?**
- run `yarn start:dev` on the terminal  to open the web browser on port 3000
- Navigate to the settings page
- Click on centers and you should see a modal popup

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#159348904](https://www.pivotaltracker.com/story/show/159348904
)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-08-04 at 5 27 59 am" src="https://user-images.githubusercontent.com/28359346/43672578-dab346c4-97a8-11e8-8347-f2839a968160.png">

**Questions:**
N/A
